packages/main/cypress/specs/AINotice.cy.ts

```diff
@@ -0,0 +1,60 @@
+import "../../src/AINotice.js";
+import "../../../icons/src/AllIcons.js";
```

Comment:
You don't need the allIcons import as you don't have tests that use so many icons.
you only check if some internal links has the "ai" icon.

If you import the "ai" within the Notice component
via
import "@ui5/webcomponents-icons/dist/ai.js";

you won't need this import at all.